dspace/config/spring/api/external-services.xml

```diff
@@ -92,5 +94,38 @@
         </property>
     </bean>
 
+    <bean id="scopusLiveImportDataProvider" class="org.dspace.external.provider.impl.LiveImportDataProvider">
+        <property name="metadataSource" ref="ScopusImportService"/>
+        <property name="sourceIdentifier" value="scopus"/>
+        <property name="recordIdMetadata" value="dc.identifier.scopus"/>
```

Comment:
I don't think we have a `dc.identifier.scopus` field in our DSpace schemas.  Do we need to add one?

Reply:
Feedback here wasn't addressed. I don't think this metadata field exists in DSpace out-of-the-box.